python/xorbits/_mars/services/cluster/file_logger.py

```diff
@@ -27,8 +28,12 @@ class FileLoggerActor(mo.Actor):
     """
 
     def __init__(self):
-        file_path = os.environ.get(MARS_LOG_PATH_KEY)
-        self._log_filename = file_path
+        log_dir = os.environ.get(MARS_LOG_DIR_KEY)
+        self._log_file_path: Optional[str] = None
+        if log_dir is None:
```

Comment:
What's the situation for log_dir is None?

Reply:
Normally, this env var is set when a supervisor or worker pool is created. 

But some test cases just use an actor pool for convenience, in which case the env var `MARS_LOG_DIR` is None.